index.html

```diff
@@ -300,85 +300,14 @@ <h3>
         </h3>
         <p>
           The {{Document/onvisibilitychange}} attribute is an <a>event handler
-          IDL attribute</a> for the `"visibilitychange"` event type (see
-          [[[#reacting-to-visibilitychange]]]).
+          IDL attribute</a> for the `"visibilitychange"` event type.
+        </p>
+
+        <p class="note">
+          The `"visibilitychange"` event is invoked from the [[HTML]] spec.
```

Comment:
I think we want: 

```suggestion
          The "[=Document/visibilitychange=]" event is invoked from the [[HTML]] spec.
```

And then over on the HTML spec, we need to define and export visibilitychange